### PR TITLE
[CWS] Remove security dump on comms and related APIs

### PR DESCRIPTION
**Pull request by spikat**
### What does this PR do?

This PR removes code that was never used 

### Motivation

Ease future code evolution/maintenance, cleanup

### CODE REVIEW

**Review by hush-hush — APPROVED** [review]
Looks good for the files owned by ASC

File: pkg/proto/pbgo/core/workloadmeta.pb.go [hunk 1]
```diff
@@ -1,7 +1,7 @@
 // Code generated by protoc-gen-go. DO NOT EDIT.
 // versions:
-// 	protoc-gen-go v1.34.0
-// 	protoc        v5.26.1
+// 	protoc-gen-go v1.28.1
```
Comment (paulcacheux) [inline pkg/proto/pbgo/core/workloadmeta.pb.go:3]: yeah I'm not sure we should touch those files

Comment (spikat) [reply pkg/proto/pbgo/core/workloadmeta.pb.go]: Indeed, thanks! They were regenerated by mistake, I've removed them in my last commit

**Review by paulcacheux — APPROVED** [review]

**Review by usamasaqib — APPROVED** [review]

**Pull request merged**